moveit_core/collision_detection_fcl/include/moveit/collision_detection_fcl/collision_common.h

```diff
@@ -166,42 +183,48 @@ struct DistanceData
   {
   }
 
-  /// Distance query request information
+  /** \brief Distance query request information. */
   const DistanceRequest* req;
 
-  /// Distance query results information
+  /** \brief Distance query results information. */
   DistanceResult* res;
 
-  /// Indicates if distance query is finished.
+  /** \brief Indicates if distance query is finished. */
   bool done;
 };
 
 MOVEIT_STRUCT_FORWARD(FCLGeometry);
 
+/** \brief Bundles the \e CollisionGeometryData and FCL collision geometry representation into a single class. */
 struct FCLGeometry
 {
   FCLGeometry()
   {
   }
 
+  /** \brief Constructor for a robot link. */
   FCLGeometry(fcl::CollisionGeometryd* collision_geometry, const robot_model::LinkModel* link, int shape_index)
     : collision_geometry_(collision_geometry), collision_geometry_data_(new CollisionGeometryData(link, shape_index))
   {
     collision_geometry_->setUserData(collision_geometry_data_.get());
   }
 
+  /** \brief Constructor for an attached body. */
   FCLGeometry(fcl::CollisionGeometryd* collision_geometry, const robot_state::AttachedBody* ab, int shape_index)
     : collision_geometry_(collision_geometry), collision_geometry_data_(new CollisionGeometryData(ab, shape_index))
   {
     collision_geometry_->setUserData(collision_geometry_data_.get());
   }
 
+  /** \brief Constructor for world object. */
```

Comment:
```suggestion
  /** \brief Constructor for a world object. */
```
Total nitpick for consistency.

Reply:
done